lib/pages/interpretation_page.dart

```diff
@@ -0,0 +1,237 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_blue/flutter_blue.dart';
+import 'package:lsa_gloves/connection/ble/bluetooth_backend.dart';
+import 'package:lsa_gloves/navigation/navigation_drawer.dart';
+
+import 'dart:developer' as developer;
+
+class InterpretationPage extends StatefulWidget {
+  const InterpretationPage({Key? key}) : super(key: key);
+
+  @override
+  State<InterpretationPage> createState() => _InterpretationPageState();
+}
+
+class _InterpretationPageState extends State<InterpretationPage> {
+  static final String appBarTitle = 'Interpretación';
+  static final Color background = Color.fromRGBO(0xD3, 0xD3, 0xD3, 1.0);
```

Comment:
Es preferible definir el color a partir del Theme.of(context) porque en caso de despues cambiar el tema el color se cambiaria automaticamente

Reply:
Done.